src/components/error-pages/common-error-page.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useMemo } from 'react'
+import type { PropsWithChildren } from 'react'
+import { LandingLayout } from '../landing-layout/landing-layout'
+import { useTranslation } from 'react-i18next'
+
+export interface CommonErrorPageProps {
+  title: string
+  description?: string
+  titleIsI18nKey?: boolean
+  descriptionIsI18nKey?: boolean
```

Comment:
Error pages should NEVER be untranslated because they provide user output. Therefore I suggest that these parameters get removed.

Reply:
Changed